src/app.rs

```diff
@@ -115,6 +115,7 @@ pub fn build_app() -> App<'static, 'static> {
                 .alias("dereference")
                 .overrides_with("follow"),
         )
+        .arg(arg("same-file-system").long("same-file-system"))
```

Comment:
ripgrep calls this option `--one-file-system`. I think we should use the same name.

Reply:
Changed for both CLI and internal options.